tests/sklearn/test_sklearn_autolog.py

```diff
@@ -819,11 +818,11 @@ def train_cv_model():
     assert isinstance(cv_model, cv_class)
 
     # Ensure that a signature and input example are produced for the best estimator
-    best_estimator_conf = get_model_conf(run.info.artifact_uri, "best_estimator")
+    model_uri = _get_model_uri(name="best_estimator")
+    best_estimator_conf = Model.load(model_uri)
```

Comment:
This test produces two models. The `last_logged_model` approach doesn't work. We can use `search_logged_models`.